app/src/utils/services/DataService.ts

```diff
@@ -3,7 +3,7 @@ import QueryBuilder, { QueryObject } from 'es-query-builder';
 import Location from '@/models/location';
 
 class DataService {
-    private baseUrl = 'https://cai-einander-helfen.fbi.h-da.de/api/_search/';
+    private baseUrl = searchURI;
```

Comment:
do we want to keep the class member now that we inject the uri ? 

an argument against it would be mutability of the baseUrl at runtime versus just always using what was injected at build time

Reply:
I think the current approach could be helpful if we need to refactor this in the future.
Now we have one place where the injected variable is being used.